web/src/compose_validate.ts

```diff
@@ -795,3 +800,124 @@ export function convert_mentions_to_silent_in_direct_messages(
     const silent_mention_text = people.get_mention_syntax(full_name, user_id, true);
     return silent_mention_text;
 }
+
+export function is_split_messages_enabled(): boolean {
+    return split_messages_enabled;
+}
+
+export function set_split_messages_enabled(enable: boolean): void {
+    split_messages_enabled = enable;
+}
+
+/**
+ * @param content
+ * @returns index of the delimiter occurring outside code blocks since no splitting should
+ * be done inside code blocks
+ */
```

Comment:
This docstrings are hard to read, and don't really seem helpful. Like, the fact that it takes one parameter, `content`, is already clarified by the type... and basically the rest of the information is already there in the function name. I think you should just delete them.

It's also not a pattern we use in this codebase, in large part because it's noisy and we're not a library, where having auto-generated documentation would be useful.

Reply:
Understood, I've removed the docstrings from self explanatory functions.